CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,39 @@
 
 import Foundation
 
-print("Hello, World!")
+func generateLotteryNumber() -> Set<Int> {
+    var lotteryNumbers: Set<Int> = []
+
+    while lotteryNumbers.count < 6 {
+        lotteryNumbers.insert(Int.random(in: 1...45))
+    }
+
+    return lotteryNumbers
+}
+
+func compareLotteryNumber(myNumbers: Array<Int>, lotteryNumbers: Set<Int>) {
+    let bothMineAndLottery = lotteryNumbers.intersection(myNumbers).sorted()
+    
+    if bothMineAndLottery.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    } else {
+        print("축하합니다! 겹치는 번호는", terminator: " ")
+        printAllElementsOf(bothMineAndLottery)
+        print("입니다!")
+    }
+}
+
+func printAllElementsOf(_ array: Array<Int>) {
```

Comment:
좋은 네이밍이지만 저는 아규먼트 레이블을 조금 더 활용할거 같아요!
이렇게하면 호출부에서도 더 깔끔해보일거 같죠?
정답은 아닙니다!

```suggestion
func printAllElements(of array: Array<Int>) {
```

Reply:
오호 감사합니다!! 더 깔끔해보이네요😲👍